uiowa/uiowa
# UIowa.edu search/menu changes

Resolves https://github.com/uiowa/uiowa/issues/4716. 

# Remaining tasks 
- [x] Browserstack testing, especially safari mobile menu toggle. 
- [x] Attach UIDS main hash once approved
- [ ] Figure out best way to deploy these changes


# How to test

`blt frontend`
`ddev blt sync --site uiowa.edu --define drush.aliases.remote=home.dev`
- Test user story "As a prospective student, I can use horizontal navigation on uiowa.edu while still having access to the links that were available in the toggle navigation"
- `ddev blt ds --site=uiowa.edu`
- Test browser breakpoints - menu should collapse at 980px and under.

## Once approved
- Attach UIDS Release from merged https://github.com/uiowa/uids/pull/784. 

## Once deployed in a release

1. "About Iowa" changes to "About" and "News, Events, Media Resources" moves under "About"
2. "Athletics", "Alumni and Friends", and "Health Care" links to need to be added to the end of the main nav
3. Menu setting on uiowa.edu needs to be set to horizontal `admin/appearance/settings/uids_base`
